### DISCUSSION TITLE
v0.0.13

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#461] [Versions & Releases]
# :frog: v0.0.13
## 💾 Code updates
- `SpeakerManager` class for handling multi-speaker model management and interfacing `speaker.json` file. 
- Enabling multi-speaker models with `tts` and `tts-server` endpoints. (:crown: @kirianguiller )
- Allow choosing a different `noise scale` for GlowTTS at inference. 
- Glow-TTS updates to import SC-Glow Models.
- Fixing windows support (:crown: @WeberJulian )

## :walking_woman:  Operational Updates
- Refactoring :frog: TTS installation and allow selecting different scopes (`all, tf, notebooks`)for installation depending on the specific needs. 

## 🚀 New Pre-Trained Model Releases
- SC-GlowTTS multi-speaker English model from our work https://arxiv.org/abs/2104.05557 (:crown: @Edresson )
- HiFiGAN vocoder finetuned for the above model. 
- Tacotron DDC Non-Binary English model using Accenture's Sam dataset. 
- HiFiGAN vocoder trained for the models above.

# Released Models
:bulb:  All the models below are available by ```tts``` or ```tts-server``` endpoints on CLI as explained [here](https://github.com/coqui-ai/TTS#example-synthesizing-speech-on-terminal-using-the-released-models).

Models with ✨️ below are new with this release.

| Language | Dataset | Model Name | Model Type | TTS version | Download|
| -- | -- | -- | -- | -- | :-: |
:sparkles: English  (non-binary)       | sam (acccenture) |  Tacotron2-DDC  | tts  | :smile: v0.0.13 | [:floppy_disk: ](https://github.com/coqui-ai/TTS/releases/download/v0.0.13/tts_models--en--sam--tacotron_DDC.zip)  
:sparkles: English  (multi-speaker)   | VCTK        | SC-GlowTTS        | tts       | :smile: v0.0.13| [:floppy_disk: ](https://github.com/coqui-ai/TTS/releases/download/v0.0.12/tts_models--en--vctk--sc-glowtts-transformer.zip)| 
English                     |  LJSpeech | Tacotron-DDC          | tts       | v0.0.12| [:floppy_disk: ](https://github.com/coqui-ai/TTS/releases/download/v0.0.12/tts_models--en--ljspeech--tacotron2-DDC.zip)
German                    | Thorsten-DE   | Tacotron-DCA     | tts       | v0.0.11   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.11/tts_models--de--thorsten--tacotron2-DCA.zip)
German                    | Thorsten-DE   | Wavegrad     | vocoder      |v0.0.11   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.11/vocoder_models--de--thorsten--wavegrad.zip)
English                    | LJSpeech   | SpeedySpeech      | tts       | v0.0.10   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.10/tts_models--en--ljspeech--speedy-speech-wn.zip)
English                    | EK1        | Tacotron2         | tts       |v0.0.10   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.10/tts_models--en--ek1--tacotron2.zip)
Russian                    | RUSLAN     | TacotronDDC       | tts       | v0.0.10   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.10/tts_models--ru--ruslan--tacotron2-DDC.zip)
Dutch                      | MAI        | TacotronDDC       | tts       | v0.0.10   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.10/tts_models--nl--mai--tacotron2-DDC.zip)
Chinese                    | Baker      | TacotronDDC-GST   | tts       | v0.0.10   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.10/tts_models--zh-CN--baker--tacotron2-DDC-GST.zip)
English                    | LJSpeech   | TacotronDCA       | tts       |v0.0.9             |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.9/tts_models--en--ljspeech--tacotron2-DCA.zip)
English                    | LJSpeech   | Glow-TTS          | tts       |v0.0.9             |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.9/tts_models--en--ljspeech--glow-tts.zip)
Spanish                    | M-AILabs   | TacotronDDC       | tts       |v0.0.9             |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.9/tts_models--es--mai--tacotron2-DDC.zip)
French                     | M_AILabs   | TacotronDDC       | tts       |v0.0.9             |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.9/tts_models--fr--mai--tacotron2-DDC.zip)
Dutch                      | MAI        | TacotronDDC       | tts       | v0.0.10   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.10/tts_models--nl--mai--tacotron2-DDC.zip)
:sparkles: English   | sam (accenture)   | HiFiGAN          | vocoder   | :smile: v0.0.13| [:floppy_disk: ](https://github.com/coqui-ai/TTS/releases/download/v0.0.13/vocoder_model--en--sam--hifigan_v2.zip) 
:sparkles: English   | VCTK       | HiFiGAN          | vocoder   | :smile: v0.0.13| [:floppy_disk: ](https://github.com/coqui-ai/TTS/releases/download/v0.0.12/vocoder_model--en--vctk--hifigan_v2.zip) 
English   | LJSpeech | HiFiGAN          | vocoder   | v0.0.12| [:floppy_disk: ](https://github.com/coqui-ai/TTS/releases/download/v0.0.12/vocoder_model--en--ljspeech-hifigan_v2.zip)
English                    | EK1        | WaveGrad          | vocoder   | v0.0.10   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.10/vocoder_models--en--ek1--wavegrad.zip)
Dutch                      | MAI        | ParallelWaveGAN   | vocoder   | v0.0.10   |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.10/vocoder_models--nl--mai--parallel-wavegan.zip)
English                    | LJSpeech   | MB-MelGAN         | vocoder   |v0.0.9             |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.9/vocoder_models--en--ljspeech--mulitband-melgan.zip)
:earth_africa:  Multi-Lang | LibriTTS   | FullBand-MelGAN   | vocoder   |v0.0.9             |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.9/vocoder_models--universal--libri-tts--fullband-melgan.zip)
:earth_africa:  Multi-Lang | LibriTTS   | WaveGrad          | vocoder   |v0.0.9             |[:floppy_disk:](https://github.com/coqui-ai/TTS/releases/download/v0.0.9/vocoder_models--universal--libri-tts--wavegrad.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.0.13'>v0.0.13</a>.</em>

**Comment by iprovalo** [comment]
@erogol I tried downloading the Russian model: https://github.com/coqui-ai/TTS/releases/download/v0.0.10/tts_models--ru--ruslan--tacotron2-DDC.zip I get "Not Found" error.

**Reply by erogol** [reply]
It is removed due to license issue
